editor/src/components/inspector/sections/layout-section/self-layout-subsection/gigantic-size-pins-subsection.tsx

```diff
@@ -49,11 +49,7 @@ export const PinsLayoutNumberControl = betterReactMemo(
   (props: PinsLayoutNumberControlProps) => {
     const framePoint = framePointForPinnedProp(props.prop)
     const pointInfo = useInspectorLayoutInfo(props.prop)
-    const fullFrame = Utils.defaultIfNull(
-      {} as FullFrame,
-      Utils.optionalMap(getFullFrame, props.frame),
```

Comment:
is `props.frame` used for other things too? if not, please also remove it from props to declutter the code

Reply:
good idea, fixed